test/ProfilerTest.cpp

```diff
@@ -1,8 +1,105 @@
 #include "gtest/gtest.h"
 
 #include "pipert/Profiler.h"
-
+#include "pipert/Scheduler.h"
+#include "pipert/Profiler.h"
+#include <cstdint>
+#include <arpa/inet.h>
+#include <sys/types.h>
+#include <unistd.h>
```

Comment:
Please only use headers which are in fact needed for this implementation. It means that you should check the relevant standards and put the headers here according to them... so do not rely on your current platform.